frontend/src/Components/Endpoint/index.module.scss

```diff
@@ -0,0 +1,46 @@
+@import "~plaid-threads/scss/variables";
+
+.endpointContainer {
+  display: grid;
+  grid-template-columns: 15% 57% 28%;
+  width: 100%;
+  border: 1px solid $black200;
+  border-bottom: 0px;
+}
+
+.post {
+  margin: 2 * $unit 0px 2 * $unit 3 * $unit;
+  font-size: 1.4rem;
```

Comment:
do we need to override the font size of the Note? it should be fine out of the box

Reply:
done